combinationSum.py

```diff
@@ -0,0 +1,29 @@
+"""
+Time Complexity : combinationSum() - O(n*2^t) where n is the number of elements in an array and t is target
+Space Complexity : combinationSum() - O(1) [O(n*2^t) - stack space]
+Did this code successfully run on Leetcode : Yes
+Any problem you faced while coding this : none
```

Comment:
How is your time complexity O(n*2^t)?

Reply:
The time complexity is correct @neha79091 . We had a separate discussion on this

Reply:
ok